concepts/placement.md

```diff
@@ -36,18 +36,38 @@ Notice that while the first example uses an `on[*]` block to tell Loci about the
 The syntax for variables is quite similar to the one for functions:
 
 ```scala
-val chats = on[Registry] { new Array[Strings](10)}
+val chats = on[Registry] { new Array[String](10)}
 ```
 
 ```scala
-val chats: Array[Strings] on Registry = placed { new Array[Strings](10)}
+val chats: Array[String] on Registry = placed { new Array[String](10)}
+```
+
+## Subjective Values
+By default, every peer accessing a shared value will receive the same data. In some cases, it is beneficial to return a different value based on the accessing peer instance. In this case subjective values can be used.
+
+The `sbj` modifier binds an identifier holding a reference to the peer instance that accesses the subjective value. This identifier can be used to filter the returned value on a per remote peer basis.
+
+For example, in the following code snippet, a `Client` accessing `publicMessage` will receive all `message` values of other `Client` instances, but not his own.
```

Comment:
explain the following code a bit more:  `sbj` creates a value on a per remote basis, the fact the clients receive messages but their own is due to the filter `if remote != client`